src/app/widgets/Location/DisplayPanel.jsx

```diff
@@ -400,13 +421,15 @@ export default connect((store) => {
     const activeState = get(store, 'controller.state.status.activeState');
     const canHome = isConnected && [GRBL_ACTIVE_STATE_IDLE, GRBL_ACTIVE_STATE_ALARM].includes(activeState) && workflowState !== WORKFLOW_STATE_RUNNING;
     const mpos = get(store, 'controller.mpos');
+    const firmware = get(reduxStore.getState(), 'controller.type');
```

Comment:
Again no point in using `reduxStore.getState()` here, especially since this is in an actual connect HOC, you already have access to the redux state from the `store` variable here